libs/src/services/creatorNode/CreatorNode.ts

```
@@ -533,23 +549,24 @@ export class CreatorNode {
     // this does the actual validation before sending to the creator node
     // if validation fails, validate() will throw an error
     try {
-      this.schemas?.[trackSchemaType].validate?.(metadata)
+      this.validateTrackSchema(metadata)
+
+      const { data: body } = await this._makeRequest(
+        {
+          url: '/tracks/metadata',
+          method: 'post',
+          data: {
+            metadata,
+            sourceFile
+          }
+        },
+        true
+      )
+      return body
     } catch (e) {
       console.error('Error validating track metadata', e)
```

confused by existing code... wouldn't the `catch` suppress the error?